index.js

```diff
@@ -51,7 +52,14 @@ let wrapCommand = function (instance) {
         if (commandGroup[fnName] && !forceOverwrite) {
             throw new Error(`Command ${fnName} is already defined!`)
         }
-        commandGroup[fnName] = fn
+        commandGroup[fnName] = function() {
+            const name = namespace ? `${namespace}.${fnName}` : fnName
+            instance.commandList.push({
+                name: name,
+                args: arguments
+            })
+            fn.apply(instance, arguments);
```

Comment:
@christian-bromann Is `instance` the correct `thisArg` for this call? Or is `null` more appropriate?